qemu/tests/block_hotplug.py

```diff
@@ -121,33 +123,43 @@ def get_new_disk(disk1, disk2):
 
             device_list.append(device)
             if not new_disks:
-                raise exceptions.TestFail("Cannot find new disk after hotplug.")
+                test.fail("Cannot find new disk after hotplug.")
 
             if params.get("need_plug") == "yes":
                 disk = new_disks[0]
             else:
                 disk = new_disks[num]
 
+            session = vm.wait_for_login(timeout=timeout)
+            if params.get("format_disk", "no") == "yes":
+                if i == 0:
+                    error_context.context("Format disk", logging.info)
+                    utils_misc.format_windows_disk(session,
+                                                   disk_index[num],
+                                                   mountpoint=disk_letter[num])
             error_context.context("Check block device after hotplug.",
                                   logging.info)
-            if params.get("pci_test_cmd"):
-                if params.get("os_type") == "linux":
-                    test_cmd = params.get("pci_test_cmd") % (disk, disk)
-                else:
-                    test_cmd = re.sub("PCI_NUM", "%s" % (num + 1),
-                                      params.get("pci_test_cmd"))
-                session = vm.wait_for_login(timeout=timeout)
-                s, o = session.cmd_status_output(test_cmd, timeout=disk_op_timeout)
-                session.close()
-                if s:
-                    raise exceptions.TestFail("Check for block device failed "
-                                              "after hotplug, Output: %r" % o)
+            if params.get("os_type") == "linux":
+                test_cmd = params.get("disk_op_cmd") % (disk, disk)
```

Comment:
could you show me where define the 'disk_op_cmd'?

Reply:
Hi coli, 
Due to here need run disk r/w operation on the new hotpluged disk, so changed the name pci_test_cmd to disk_op_cmd, has updated in the cfg file.

Reply:
I think it is better to add \"if params.get(\"disk_op_cmd\"):\" above \"if params.get(\"os_type\") == \"linux\":\"